glhe/utilities/functions.py

```diff
@@ -492,7 +492,7 @@ class Interpolator1D(InterpolatorBase):
 
     def __init__(self, x_data: np.ndarray, y_data: np.ndarray | list[float]):
         """
-        1D Interpolation Class, currently a wrapper for scipy interpolator, but soon just a simple interpolator
+        1D Interpolation Class, currently a wrapper for sci py interpolator, but soon just a simple interpolator
```

Comment:
🙈

Reply:
Yeah, I didn't want to add scipy to my spelling list, so just split the word.  Hopefully soon we won't have to mention it at all...